book/requirements/useCases/details/mainSuccessScenario.md

```diff
@@ -1,43 +1,59 @@
+ <link rel="stylesheet" href="{{baseUrl}}/book/requirements/useCases/css/useCases.css">
+
 The Main Success Scenario (MSS) describes the most straightforward interaction for a given use case, which assumes that nothing goes wrong. This is also called the _Basic Course of Action_ or the _Main Flow of Events_ of a use case. Given below is another example of an MSS.
 
 <tip-box>
+	<div>
+		<ul>
+			<li>System: Online Banking System (OBS)</li>
+			<li> Use case: UC23 - Transfer Money </li>
+			<li> Actor: User </li>
+			<li> MSS:
+				<ol>
+					<li>User chooses to transfer money.</li>
+					<li>OBS requests for details of the transfer.</li>
+					<li>User enters the requested details.</li>
+					<li>OBS requests for confirmation.</li>
+					<li>OBS transfers the money and displays the new account balance.</li>
+					<li class="custom-bullet-point">Use case ends.</li>
+				</ol>
+			</li>
+			<li> Extensions:
+				<ol>
+					<li class="custom-bullet-point">3a. OBS detects an error in the entered data.
```

Comment:
@acjh is there a way to avoid repeating `class="custom-bullet-point"` in every list item?